### ISSUE TITLE
compilation failure in ypaint.cc due to missing include

### ISSUE THREAD

**Issue by hannob** [bbidulock/icewm#329]
ypaint.cc uses the mblen feature, however it doesn't include the right headers for it.

This can lead to compilation errors:
```
ypaint.cc: In member function ‘void Graphics::drawStringEllipsis(int, int, const char*, int)’:
ypaint.cc:269:24: error: ‘mblen’ was not declared in this scope
         if (multiByte) mblen(NULL, 0);
                        ^~~~~
```

This only appears if freetype support is disabled, because otherwise the include will indirectly happen through Xft.h. However relying on indirect includes is bad.

Adding an
```
#include <cstdlib>
```
fixes this. I'd do a pull request, but I'm unsure which is the right branch for current development (latest versions are 1.5.x, yet the main branch seems to be an 1.4-one.)

**Comment by gijsbers** [comment]
@hannob Thanks! Could you let me know which operating system (+ version number) you are compiling on as I could not reproduce this problem.

**Comment by hannob** [comment]
Gentoo, icewm 1.5.1. (It may be that this only shows up with a recent glibc, sometimes they make the header include things more strict, I have glibc 2.28.)

Reproduce with
```
./configure --disable-xfreetype
make

**Comment by hannob** [comment]
fix should be fine, 1.5.5 no longer has this issue
